charts/ndb-operator/README.md

```diff
@@ -98,7 +98,7 @@ spec:
       compute:
         id: ""
         name: ""
-      # A Software profile is a mandatory input for closed-source engines: MSSQL & Oracle
+      # A Software profile is a mandatory input for closed-source engines: MSSQL
```

Comment:
why no more oracle ?

Reply:
We don't support oracle. This was a mistake in the ReadMe.